### ISSUE TITLE
Archives

### ISSUE THREAD

**Issue by lukasz4444** [ShicKla/AuspexGateSystems#15]
Please add **archives** to ags for sorting and please add so every archive can have **own password**.

**Comment by ShicKla** [comment]
Please clarify what you mean by archives. Are you imagining something like a separate folders to keep addresses in, and having some of those folders protected?

**Comment by lukasz4444** [comment]
yes. i mean folder directories etc that user can create with password that user can set.
